src/resmom/linux/mom_mach.c

```diff
@@ -129,6 +129,13 @@ static char	procfs[] = "/proc";
 static DIR	*pdir = NULL;
 static int	pagesize;
 static long	hz;
+
+/* convert between jiffies and seconds */
+#define	JTOS(x)	(((x) + (hz/2)) / hz)
```

Comment:
Here's what the C99 standard (section 6.7.8, Initialization) says about static initializations:

If an object that has automatic storage duration is not initialized explicitly, its value is
indeterminate. If an object that has static storage duration is not initialized explicitly,
then:
- if it has pointer type, it is initialized to a null pointer;
- if it has arithmetic type, it is initialized to (positive or unsigned) zero;
- if it is an aggregate, every member is initialized (recursively) according to these rules;
- if it is a union, the first named member is initialized (recursively) according to these
rules.